todoTask/todoTask.js

```diff
@@ -1,85 +1,59 @@
-﻿// ------------------------global varibals--------------------
-let globalID = 0;
-// -------------------------functions---------------------------
-const getId = () => (globalID += 1);
+const getId = (() => {
+    let globalID = 0;
+    return () => (globalID += 1);
+})();
 
-const makeTodo = (title, complete) => {
-    return {
-        id: getId(),
-        title,
-        complete,
-    };
-};
+const createTodo = (title, complete = false) => ({
+    id: getId(),
+    title,
+    complete,
+});
 
-const makeTodos = (n) => {
-    let todos = [];
-    for (let i = 1; i <= n; i++) {
-        let title = `title${i}`;
-        let complete = false;
-        let todo = makeTodo(title, complete);
-        todos = addTodo(todos, todo);
-    }
-    return todos;
-};
+const createTodos = (count) =>
+    Array.from({ length: count }, (value, i) => createTodo(`title${i}`));
```

Comment:
That's a native way for generating arrays. It can be considered as "too smart", so use it carefully or move it to generic `times` util function